agent/acs/handler/acs_handler.go

```diff
@@ -339,7 +346,7 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 
 	client.AddRequestHandler(payloadHandler.handlerFunc())
 
-	heartbeatHandler := newHeartbeatHandler(acsSession.ctx, client, acsSession.dockerClient, acsSession.agentConfig.Cluster, acsSession.containerInstanceARN)
+	heartbeatHandler := newHeartbeatHandler(acsSession.ctx, client, acsSession.doctor)
```

Comment:
I would prefer that we pass the `cluster` and `containerInstanceARN` as separate parameters rather than combining them in Doctor.

Reply:
We only need cluster and container instance arn for the doctor, they're not being used otherwise.